.storybook/SidebarLabel/index.styles.tsx

```diff
@@ -0,0 +1,37 @@
+import pixelsToRem from '../../src/app/utilities/pixelsToRem';
+
+const styles = {
+  iconWrapper: {
+    display: 'inline-block',
+    boxSizing: 'border-box',
+    marginLeft: `0.5em`,
```

Comment:
em?

Reply:
Webcore uses em, I am assuming it was a UX requirement on their side to have the spacing depending on font size.